src/pybind/mgr/CMakeLists.txt

```diff
@@ -1,6 +1,23 @@
+set(mgr_module_install_excludes
+  PATTERN "CMakeLists.txt" EXCLUDE
+  PATTERN ".gitignore" EXCLUDE
+  PATTERN "tox.ini" EXCLUDE
+  PATTERN "requirements*.txt" EXCLUDE
+  PATTERN "constraints*.txt" EXCLUDE
+  PATTERN "tests" EXCLUDE)
+
 if(WITH_MGR_DASHBOARD_FRONTEND)
   add_subdirectory(dashboard)
 endif()
+
+install(DIRECTORY dashboard
+  DESTINATION ${CEPH_INSTALL_DATADIR}/mgr
+  ${mgr_module_install_excludes}
+  PATTERN "frontend/node_modules" EXCLUDE
+  PATTERN "frontend/cypress*" EXCLUDE
+  PATTERN "frontend/src" EXCLUDE
+  PATTERN ".*" EXCLUDE)
```

Comment:
Have you checked if `PATTERN` worked as expected here? I think it basically converted the `pattern*` to something like `.*/pattern[^/]*`. So I'm not sure if `frontend/cypress*` will match `frontend/cypress/integration/*` for example... Did it work?

Reply:
@epuertat good point. i am still testing it at https://shaman.ceph.com/builds/ceph/wip-kefu-testing-2021-07-12-2221/4e929ee2ab4e5c32aefa3ddec1f86e6c27f4fd99/

Reply:
will fix it.